src/workloads/scale/LLTInsert.yml

```diff
@@ -25,22 +25,31 @@ GlobalDefaults:
   # Loader Config.
   LoadThreads: &LoadThreads 4
   LoadBatchSize: &LoadBatchSize 1000
+
+  # In-memory
+  InitialDocumentCount: &InitialNumDocs 10000000
   SecondaryDocumentCount: &SecondaryNumDocs 10000000
+
+  # Not In-memory
+  #InitialDocumentCount: &InitialNumDocs 49000000
+  #SecondaryDocumentCount: &SecondaryNumDocs 1000000
   CollectionCount: &CollectionCount 4
-  InitialDocumentCount: &InitialNumDocs 10000000
 
   # Common Baseline / Benchmark
   InsertBatchSize: &InsertBatchSize 50
 
   # Low Baseline / Benchmark
-  LowGlobalRate: &LowGlobalRate 10% # percentage gives a smoother rate. 400 per 1 second was the other option.
+  LowGlobalRate: &LowGlobalRate 400 per 1 second
   LowThreads: &LowThreads 4
-  LowDuration: &LowDuration 1 minutes
 
   # Low Baseline / Benchmark
-  HighGlobalRate: &HighGlobalRate 50% # percentage gives a smoother rate. 3200 per 1 second was the other option.
+  HighGlobalRate: &HighGlobalRate 800 per 1 second
   HighThreads: &HighThreads 8
-  HighDuration: &HighDuration 1 minutes
+
+  # Scanner
+  SnapshotScanner2MinutesDuration: &SnapshotScanner2MinutesDuration 2 minutes
```

Comment:
Um, these anchors are longer than the text they are replacing. Is there a reason why you want to use an anchor here?